e2e-test/src/test/java/de/sovity/edc/extension/e2e/connector/config/EdcApiType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *      sovity GmbH - init
+ */
+
+package de.sovity.edc.extension.e2e.connector.config;
+
+public enum EdcApiType {
```

Comment:
Nit: Why doesn't ApiConfig use this

Reply:
Reused the `EdcApiType` class in `ApiConfig`